__tests__/app.test.js

```diff
@@ -318,4 +318,28 @@ describe("DELETE: /api/comments/:comment_id", () => {
       })
     })
   })
+})
+
+describe("GET /api/users", () => {
+  test("200: Responds with an array of users", () => {
+    return request(app)
+    .get('/api/users')
+    .expect(200)
+    .then(({body}) => {
+      const {users} = body;
+      users.forEach((user) => {
+        expect(user).toMatchObject({
+          username: expect.any(String),
+          name: expect.any(String),
+          avatar_url: expect.any(String)
+        })
+      })
+    })
+  })
+
+  test("404: Responds with: 'Users not found' when a typo is made", () => {
+    return request(app)
+    .get('/api/userss')
+    .expect(404)
+  })
```

Comment:
So if you have already somewhere tested for an incorrect endpoint/mistyped then you don't need to do it again. 

One test for an endpoint that doesn't match is fine across the whole project, it should also live in it's own describe block as it is not linked to /api/users for example. 

No errors are needed for this endpoint!